### ISSUE TITLE
`twisted.internet.endpoints.connectProtocol` is missing @since

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6610]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#6610|
|Type|defect|
|Created|2013-07-03 13:42:35Z|

Attachments:

* [issue#6610.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/b5d/b5d4d427daafb74c01da5f875db9d87a6fac559b/463896ea2b8dcc005a7ba1b2fa3a91fc1cad5fca.patch) (499 bytes) - added by gen3 on 2013-07-07 10:37:20Z - this is my first patch. and i think this is not that simple.

<details><summary>Searchable metadata</summary>

```
trac-id__6610 6610
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1372858955000000 1372858955000000
changetime__1373566096000000 1373566096000000
version__None None
owner__therve therve

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

A great way to fix this would be to resolve [#5309](https://github.com/twisted/twisted/issues/5309).

Also, I deleted the totally unrelated "And ..." from the ticket description, sorry!  Please file different tickets for different tasks.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_4"></a>|@therve set owner to @therve|
|-|-|

Cool applying.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set status to `closed`|
|-|-|

(In [39129]) Author: gen3
Reviewer: therve
Fixes: [#6610](https://github.com/twisted/twisted/issues/6610)

Add @since marker to twisted.internet.endpoints.connectProtocol
